src/utils/lodash-mixins.ts

```diff
@@ -11,7 +11,7 @@ declare module "lodash" {
     interface LoDashImplicitWrapper<TValue> {
         getOrFail<TObject extends object, TKey extends keyof TObject>(
             this: LoDashImplicitWrapper<TObject | null | undefined>,
-            path: TKey | [TKey]
+            path: TKey | [TKey] | string
```

Comment:
This extra `| string` makes `_.getOrFail` easier to use, but loosens the type safety. The code in the caller can be adapted for the original type to work. For example, I guess your problem was here:

```
        const keys: Array<keyof DashboardData> = ["items", "charts", "reportTables"]
        const { items, charts, reportTables } = _(keys)
            .map(key => [key, _(dashboardItems).getOrFail(key)])
            .fromPairs()
            .value();
```

and here:

``` 
        const keys = ["antigenCategory", ...expectedCharts] as Array<keyof typeof dashboardItemsElements>
        const { antigenCategory, ...elements } = _(keys)
            .map(key => [key, _(dashboardItemsElements).getOrFail(key)])
            .fromPairs()
            .value();
```

Note that the first snippet is completely type-safe, while the second is not because `expression as type` is a type coercion (the real type would be very verbose, this will do)

Reply:
done!